src/CodeGeneration/ApiGenerator/Views/LowLevel/Client/Methods/MethodDocs.cshtml

```diff
@@ -16,7 +16,7 @@
 </text>
 }
 		///@Raw(@"<param name=""requestParameters"">Request specific configuration such as querystring parameters &amp; request specific connection settings.</param>")
-		@if (!string.IsNullOrEmpty(method.DeprecatedPath))
+		@if (method.DeprecatedPath != null)
 		{
-<text>		[Obsolete("@method.DeprecatedPath")]
+<text>		[Obsolete("Deprecated as of @(method.DeprecatedPath.Version): @Raw(method.DeprecatedPath.Description)")]
```

Comment:
Maybe

[Obsolete("Deprecated in version @(method.DeprecatedPath.Version): @Raw(method.DeprecatedPath.Description)")]

Reply:
Updated.